### PR TITLE
Notebook enhancement: add progress bar

### LINKED ISSUE #39: training progress bar for notebooks
The main suggestion we've gotten is to display progress bars rather than a lot of text output.

https://github.com/tqdm/tqdm#ipython-jupyter-integration

### PR DESCRIPTION
**Pull request by imyiningliu**
closes #39

### CODE REVIEW

**Comment by imyiningliu** [comment]
Do we want to add `tqdm` as a dependency?

**Comment by jeff-regier** [comment]
Yes, that's fine

**Comment by imyiningliu** [comment]
It shows a progress bar in Jupyter ;)

File: scvi/train.py [hunk 1]
```diff
@@ -5,20 +5,21 @@
...
-    stats = Stats(n_epochs=n_epochs, benchmark=benchmark)
+    stats = Stats(n_epochs=n_epochs, benchmark=benchmark, verbose=verbose)
     stats.callback(vae, data_loader_train, data_loader_test)
     early_stopping = EarlyStopping(benchmark=benchmark)
 
     # Training the model
-    for epoch in range(n_epochs):
+    for epoch in tqdm(range(n_epochs)):
         total_train_loss = 0
         for i_batch, (tensors_train) in enumerate(data_loader_train):
```
[the remark this answers is not in the document]
Comment (jeff-regier) [reply scvi/train.py]: Actually this looks pretty good as is ... I think for `brain_large` each inner loop may take so long that another progress bar would be nice, but usually just one progress bar is enough, so let's leave it as it is.

**Comment by jeff-regier** [comment]
@imyiningliu Will you update the Jupyter notebook so it shows the progress bar? Also, maybe a good time to switch from using `load_dataset("cortex")` to calling `CortexDataset()`.

**Pull request merged**